### ISSUE TITLE
Upgrade of OCP to the nightly build failing with undefined ocp_channel

### ISSUE THREAD

**Issue by petr-balogh** [red-hat-storage/ocs-ci#2572]
```           logger.info(f" oc version: {ocp.get_current_oc_version()}")
            # Verify Upgrade subscription channel:
>           ocp.patch_ocp_upgrade_channel(ocp_channel)
E           UnboundLocalError: local variable 'ocp_channel' referenced before assignment```

Job:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10304/console

**Comment by petr-balogh** [comment]
I did some fix in my WA PR here:  https://github.com/red-hat-storage/ocs-ci/pull/2555/commits/01f7735bbab29e3e4f959cdcfc68524104c1dc22

Based on logic I've checked we should define the channel in nightly config files as well to be able to upgrade from 4.3 to 4.4 nightly I think.

Only weird message is this:
```  - lastTransitionTime: "2020-07-27T12:48:08Z"
    message: 'Unable to retrieve available updates: currently installed version 4.4.0-0.nightly-2020-07-25-091418
      not found in the "stable-4.4" channel'```

Which we need to check if it's OK or not.

**Comment by AviadP** [comment]
fixed by [PR#2751](https://github.com/red-hat-storage/ocs-ci/pull/2751)
